consensus/polybft/contractsapi/gen/main.go

```diff
@@ -0,0 +1,73 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"runtime"
+
+	"github.com/0xPolygon/polygon-edge/consensus/polybft/contractsapi/artifact"
+	"github.com/dave/jennifer/jen"
+)
+
+func main() {
+	_, filename, _, _ := runtime.Caller(0) //nolint: dogsled
+	currentPath := path.Dir(filename)
+	scpath := path.Join(currentPath, "../../../../core-contracts/artifacts/contracts/")
+
+	f := jen.NewFile("contractsapi")
+	f.Comment("This is generated file don't change it manually.  DO NOT EDIT.")
```

Comment:
Nitpick: shorten this comment (initially I was triggered to change it because of one extra space before "DO NOT EDIT" part, so at least trim that extra space)? :slightly_smiling_face: 

```suggestion
	f.Comment("This is auto-generated file. DO NOT EDIT.")
```